### ISSUE TITLE
Tempo element can be selected only by clicking on note shape and equals sign, but not on tempo text

### ISSUE THREAD

**Issue by rualark** [paulrosen/abcjs#492]
Clicking on tempo text should also select tempo element.

![image](https://user-images.githubusercontent.com/15198808/88439199-34937180-ce13-11ea-9461-f53c2256484b.png)

```
%%barnumbers 1
Q:1/4=100
M:4/4
K:C
L:1/16
V: V0 clef=treble name="Sop."
V: V1 clef=treble name="Alt."
V: V2 clef=treble-8 name="Ten."
V: V3 clef=bass name="Bas."
[V: V0]"_I"c4d4e4d4|c4z12|z16|z16|
[V: V1]z16|z16|z16|z16|
[V: V2]z16|z16|z16|z16|
[V: V3]z16|z16|z16|z16|
```

**Comment by paulrosen** [comment]
Will be fixed in 6.0.0-beta.14

**Comment by rualark** [comment]
✅ tested
